### ISSUE TITLE
Payment Gateways - iFrame functionality not working

### ISSUE THREAD

**Issue by Noodleyman** [cubecart/v6#492]
Within the "public function transfer() {" function of a payment gateway, setting the submit type to be "iframe" instead of "auto" should load the content in an iFrame, however in CC6 it isn't loading the content.

iFrames in payment gateways appears to be non functional, tested with 6.0.4

**Comment by abrookbanks** [comment]
Thanks for this. Just to remind myself the PayPal Hosted iframe isn't loading.

**Comment by abrookbanks** [comment]
This does actually work but I gave you bad advice... 

I tested this with the print order form which worked if I did TWO things.

1. In the transfer function of the Gateway class set "submit" to "iframe".
2. Make sure there is a public function "iframeURL" which returns the iframe URL.

![screen shot 2015-05-26 at 11 03 54](https://cloud.githubusercontent.com/assets/4520803/7810095/2ed7c6c0-0397-11e5-899c-45490d4180e2.png)
